src/Commands/Base/PnPAdminCmdlet.cs

```diff
@@ -76,12 +73,9 @@ protected override void BeginProcessing()
                     PnPConnection.CurrentConnection.Context =
                         PnPConnection.CurrentConnection.CloneContext(adminUrl);
                 }
-                else if (PnPConnection.CurrentConnection.ConnectionType == ConnectionType.TenantAdmin)
```

Comment:
Removed this IF statement as the ConnectionType only knows 2 options: O365 and TenantAdmin. For both of these it should populate the _baseUrl in a similar fashion.